clang/test/3C/base_subdir/unwritable_rootcauses.c

```diff
@@ -0,0 +1,16 @@
+// RUN: cd %S
+// RUN: 3c -alltypes -addcr -output-dir=%t.checked/base_subdir -warn-all-root-cause %s -- -Xclang -verify=unwritable-expected
+
+#include "../unwritable_rootcauses.h"
+#include "../root_cause.c"
+
+int foo(void) { 
+  int *a = ptr;
+  int *z = a;
+  int* q = f1();
+  int *w = arr + 2;
+
+  f2(q); // unwritable-expected-warning {{1 unchecked pointer: Default void* type}}
+  
+  return *z + *q + *w;
```

Comment:
What is the purpose of this code?  To make the counts of pointers affected by the root causes in the unwritable files more interesting?  If so, please add a comment saying that.